lightwood/api/json_ai.py

```diff
@@ -151,8 +150,9 @@ def lookup_encoder(
                 encoder_dict["args"]["grouped_by"] = f"{gby}"
                 encoder_dict["args"]["timesteps"] = f"{tss.horizon}"
                 encoder_dict["module"] = "TsArrayNumericEncoder"
-        if "__mdb_ts_previous" in col_name:
-            encoder_dict["module"] = "ArrayEncoder"
+
+        if "__mdb_ts_previous" in col_name or col_name in tss.historical_columns:
+            encoder_dict["module"] = "TimeSeriesEncoder"
```

Comment:
Why use the new timeseries encoder for historical columns and target-history columns?

Reply:
Both target history and "historical columns" are time series info. that we can featurize with the new TS encoder (by normalizing and then getting a few moving averages).